ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py

```diff
@@ -3363,6 +3363,47 @@ def test_tensorflow_Prod(  # NOQA
     )
 
 
+# RandomStandardNormal
+@handle_frontend_test(
+    fn_tree="tf.raw_ops.RandomStandardNormal",
+    shape=helpers.get_shape(
+        allow_none=False,
+        min_num_dims=1,
+        max_num_dims=5,
+        min_dim_size=1,
+        max_dim_size=10,
+    ),
+    dtype=helpers.get_dtypes("float", full=False),
+    seed=helpers.ints(min_value=0, max_value=10),
+    seed2=helpers.ints(min_value=0, max_value=10),
```

Comment:
No need for max and min values
```suggestion
    seed=helpers.ints(),
    seed2=helpers.ints(),
```